src/Lobby.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Socket } from "socket.io-client";
+
+import LobbyForm from "LobbyForm";
+import React from "react";
+import { css, StyleSheet } from "aphrodite";
+import logo from "./static/logo.png";
+
+type Props = Readonly<{
+  socket: Socket;
+}>;
+
+const Lobby = ({ socket }: Props) => {
```

Comment:
rest of the codebase uses `function MyComponent(` as the syntax, lets' do that

Reply:
Renamed line 12 to `function MyComponent(` and re-uploaded the file.